pages/demo.html

```diff
@@ -17,7 +17,8 @@
   <a href="#ok" data-hotkey="o k">Press <kbd>o k</kbd> click this link</a>
 
   <script type="module">
-    import {install} from '../dist/index.js'
+    // import {install} from '../dist/index.js'
+    import {install} from 'https://unpkg.com/@github/hotkey@latest?module'
```

Comment:
I think given the change @iansan5653 just made in https://github.com/github/hotkey/pull/91 this change should no longer apply

Reply:
removed this change 6693f1f